targets/TARGET_NORDIC/TARGET_NRF5x/nrf5x_lf_clk_helper.h

```diff
@@ -40,21 +40,58 @@
     
 #ifndef MBED_CONF_NORDIC_NRF_LF_CLOCK_SRC
     #define MBED_CONF_NORDIC_NRF_LF_CLOCK_SRC (NRF_LF_SRC_XTAL)
-    #warning No configuration for LF clock source. Xtal source will be used as a default configuration.
+    #error No configuration for LF clock source. Xtal source will be used as a default configuration.
```

Comment:
Why is this error now? The text indicates that default config would be used (should not be an error?)